content/en/platform/corda/5.0/key-concepts/cordapp-dev/_index.md

```diff
@@ -12,7 +12,90 @@ section_menu: corda5
 
 # Architecture for CorDapp Developers
 
-This section describes Corda 5 from the perspective of a [CorDapp Developer]({{< relref "../../../../../about-the-docs/_index.md#cordapp-developer" >}}). Corda is an application host for CorDapps. These are pieces of code, workflows, and contracts, executed on behalf of the members of an {{< tooltip >}}application network{{< /tooltip >}}, usually as part of an interaction or {{< tooltip >}}transaction{{< /tooltip >}} between peers on the network. CorDapp Developers are responsible for designing and writing this code.
+This section describes Corda 5 from the perspective of a [CorDapp Developer]({{< relref "../../../../../about-the-docs/_index.md#cordapp-developer" >}}). Corda is an application host for {{< tooltip >}}CorDapps{{< /tooltip >}}. These are pieces of code, workflows, and contracts, executed on behalf of the members of an {{< tooltip >}}application network{{< /tooltip >}}, usually as part of an interaction or {{< tooltip >}}transaction{{< /tooltip >}} between peers on the network. CorDapp Developers are responsible for designing and writing this code.
 
-This section contains the following:
-{{< childpages >}}
+A CorDapp is distributed because parts of the application can be, or may need to be, executed on a Corda instance that is operated by another party that is a member of the same Corda {{< tooltip >}}application network{{< /tooltip >}}.
+This is not the same as an application that has multiple instances, hosted on different environments.
+
+The following diagram shows a centralized application. This is a single application that always executes within a single execution environment, regardless of the number of deployments:
+
+{{<
+  figure
+	 src="centralized-application.png"
+   width="30%"
+	 figcaption="Centralized Application"
+>}}
+
+Similarly, centralized applications often use a services architecture that means different parts of the applications interact with each other. This could be inside an entity’s network or different parts may be operated by different parties.
```

Comment:
```suggestion
Similarly, centralized applications often use a services architecture, meaning different parts of the applications interact with each other. This could be inside an entity’s network, or different parts may be operated by different parties.
```